src/shared/proto/robot_component_status.proto

```diff
@@ -0,0 +1,122 @@
+syntax = "proto3";
+
+/* Data about the status of the break beam */
+message BreakBeamStatus
+{
+    // Whether or not the ball is interrupting the robot’s laser beam.
+    bool ball_in_beam = 1;
+
+    // The reading of the robot’s laser sensor, normalized to [0,1]
+    float break_beam_reading = 2;
+}
+
+/* Data about the status of the chipper and kicker */
+message ChipperKickerStatus
+{
+    // Indicates the time elapsed since chipper was last fired in ms
+    // UINT64_MAX indicates chipper was never fired
+    uint64 time_since_chipper_fired = 1;
```

Comment:
I know using the unsigned version of whatever the largest int type available is pretty "standard" for time-based stuff, but a `uint32` would still represent nearly 50 days of time. This might be nitpicking at this point, but with the possibility of using these protos over radio (I know that's undecided atm) we may want to consider using `uint32` for saving space.

**Edit:** Although I suppose since it's encoded as a varint it doesn't matter

Reply:
Do we have a use-case for these values?

Reply:
From a previous comment

> JK I have changed my mind, I actually think the time_since_fired... values are better than autokick_fired. Assuming we use these to help determine when robots have kicked the ball (for things like when an Action is done), the time_since_fired values are better because we can report something as fired if we receive any message with the time_since value being less than some threshold, indicating the chicker was fired in the last n seconds. This makes it easy to tune on the software side.

> This is better because if we just had autokick_fired, we don't really know how long after the chicker fired that bool will be true. If we miss a few messages or the bool isn't true for very long, we may "miss" the chicker firing.

> tl;dr I say keep the time_since values and remove autokick_fired

Reply:
I've removed autokick_fired

Reply:
@akhilveeraghanta @jonathanlew @garethellis0 thoughts on `uint64` vs `uint32`? Leave it until we for sure decide to send proto over radio?

Reply:
> Leave it until we for sure decide to send proto over radio?

That sounds good

Reply:
I think that we should leave it as `uint64` given that we're using varint's (as you say) and thus there shouldn't be any size difference.